src/Kdyby/Monolog/DI/MonologExtension.php

```diff
@@ -75,7 +76,7 @@ public function loadConfiguration()
 
 		// Tracy adapter
 		$builder->addDefinition($this->prefix('adapter'))
-			->setClass('Kdyby\Monolog\Diagnostics\MonologAdapter', [$this->prefix('@logger')])
+			->setClass($config['adapter'], [$this->prefix('@logger')])
```

Comment:
AFAIK you can already do this using `services` section of neon configuration, no?